### DISCUSSION TITLE
linking-unlinking-multilingual-pages-sharepoint/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#618] [Comments]
# linking-unlinking-multilingual-pages-sharepoint/

Azure / Microsoft 365 / Visual Studio Code / Speaker / Blogger

https://www.eliostruyf.com/linking-unlinking-multilingual-pages-sharepoint/

**Comment by ollimenzel** [comment]
Hi if anyone wants to use PowerShell, I wrote about this a while back.
https://www.menzel.it/post/2023/07/reconnect_pages/
It's been talking about ShareGate but you can use it every time you copy Pages.

**Comment by Filip-BVB** [comment]
Hi Elio, 

Thank you for this information, really helpful! 

I think there is a small mistake in the code, I believe there is not supposed to be a comma after "SP.Field" in the request to disable read-only on the fields.  

Grtz, 

Filip

**Reply by estruyf** [reply]
Thanks, @Filip-BVB; the trailing comma comes from Prettier in combination with my TypeScript config. It seems the other code example also had it, so a fix is on its way.
